src/stories/Library/nav-teaser/NavTeaser.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from "react";
+
+type NavTeaserProps = {
+  title: string;
+  subtitle?: string;
+};
+
+const NavTeaser: FC<NavTeaserProps> = ({ title, subtitle }) => {
+  return (
+    <article className="nav-teaser">
+      <a href="#">
+        <h1 className="nav-teaser__title">{title}</h1>
+
+        {subtitle ? <p className="nav-teaser__subtitle">{subtitle}</p> : ""}
+
+        <svg
```

Comment:
The arrow is not growing on hover. I guess that this should happen same as everywhere else

Reply:
It's also a totally different size than all the other arrows.

Reply:
If you say that we agreed on the meeting that this specific component should not have hover behaviour, then that is fine. However, i don't remember this myself and i find it strange that this should behave differently. 

If you prefer not to, then sure, if it was me i would add it.